api/envoy/config/filter/http/compressor/v2/compressor.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package envoy.config.filter.http.compressor.v2;
+
+option java_outer_classname = "CompressorProto";
+option java_multiple_files = true;
+option java_package = "io.envoyproxy.envoy.config.filter.http.compressor.v2";
+option go_package = "v2";
+
+import "google/protobuf/wrappers.proto";
+
+import "validate/validate.proto";
+import "gogoproto/gogo.proto";
+
+// [#protodoc-title: Compressor]
+
+message Compressor {
+
+  // Minimum response length, in bytes, which will trigger compression. The default value is 30.
+  google.protobuf.UInt32Value content_length = 1 [(validate.rules).uint32.gte = 30];
```

Comment:
Why can't it be less than 30?

Reply:
Again there're no reasons for a hard limit here other than practical considerations of adding compression overhead greater than gain. E.g. Nginx defaults to 20, but doesn't insist on it.

I'd drop the limit.

Reply:
@gsagula ping. I'll drop the limit in a couple of days if you don't mind.

Reply:
Sure. Go ahead.